ballerina/http_service_endpoint.bal

```diff
@@ -19,7 +19,7 @@ import ballerina/jballerina.java;
 
 # This is used for creating HTTP server endpoints. An HTTP server endpoint is capable of responding to
 # remote callers. The `Listener` is responsible for initializing the endpoint using the provided configurations.
-public isolated class Listener {
+public class Listener {
```

Comment:
As discussed let's make it isolated.

Reply:
Fixed in [`76d3f76`](https://github.com/ballerina-platform/module-ballerina-http/pull/713/commits/76d3f763edb0eaf25424ea2ab216eba4e1184079)